apps/web/jest.config.js

```diff
@@ -0,0 +1,11 @@
+const nextJest = require("@blitzjs/next/jest")
+
+const createJestConfig = nextJest({
+  dir: "./",
+})
+
+const customJestConfig = {
+  testEnvironment: "jest-environment-jsdom",
+}
+
+module.exports = createJestConfig(customJestConfig)
```

Comment:
I assumed we want to have the same API ad `next/jest`: https://nextjs.org/docs/testing